### ISSUE TITLE
Return with multi-line strings indents strangely

### ISSUE THREAD

**Issue by jdufresne** [emacs-php/php-mode#227]
Observe the following test case:

The buffer has been indented automatically.

```
<?php


function my_other_func() {
    return "anoter really long string " .
                                        "some more text";
}

function my_func() {
    return "a really long string with = inside " .
                                      "some more text";
}
```

I would expect the second line of the strings to:
1. Indent consistently (the `=` messes with things)
2. Indent simply by `c-basic-offset`

**Comment by syohex** [comment]
> Indent consistently (the = messes with things)

I fixed and sent PR #229. Its screencast is here

![php-mode-227](https://cloud.githubusercontent.com/assets/554281/6476632/d4d32a4c-c25b-11e4-96dc-d7129c91f110.gif)

> Indent simply by c-basic-offset

Which coding style do you use ? And does it define coding style about multi line strings ?
If you don't need many spaces, you can write like this.

```php
function my_func() {
    return "a really long string with = inside "
        . "some more text";
}
```

**Comment by syohex** [comment]
@jdufresne I have merged PR #229. Please check latest version.

**Comment by cweiske** [comment]
This is fixed with the merged patch.
